client/src/app/components/Toolbar/Toolbar.scss

```diff
@@ -57,4 +60,10 @@
     top: 50%;
     transform: translate(-50%, -50%);
   }
+
+  &__form {
+    position: absolute;
+    right: 0;
+    top: 50px;
```

Comment:
the shopping list is displayed above this form add some z-index or change it in shoppinglist scss